game/src/test/java/com/hiddenswitch/spellsource/CustomCardsTests.java

```diff
@@ -3446,121 +3446,70 @@ public void testTouchOfKarma() {
 	}
 
 	@Test
-	public void testKegSmash() {
```

Comment:
Let's keep Keg Smash but mark it uncollectible and keep its test.

Reply:
Keg Smash has been renamed to Mark of Despair, but it was not read as such by GitHub for some reason.